### ISSUE TITLE
[BUG] Accessibility documentation on website doesn't have menu

### ISSUE THREAD

**Issue by regseb** [microsoft/playwright#15884]
**Describe the bug**

The menu isn't displayed in the [`Accessibility` class](https://playwright.dev/docs/api/class-accessibility) documentation page. And the menu (on the others pages) doesn't contain a link to _Accessibility_ page (I found the link with [`page.accessibility`](https://playwright.dev/docs/api/class-page#page-accessibility)).
